frontend/public/vendor.scss

```diff
@@ -26,6 +26,7 @@
 @import "~@patternfly/patternfly/utilities/Display/display";
 @import "~@patternfly/patternfly/utilities/Sizing/sizing";
 @import "~@patternfly/patternfly/patternfly-charts";
+@import "~@patternfly/patternfly/patternfly-addons";
```

Comment:
Is this for `pf-u-ml-xl-on-md`?  Are you sure it's needed.  We have other instances of `pf-u-m*` in the code base already..

Reply:
The only place in `/public/components/*` that PF spacing classes are used currently, is in cluster settings and that because it's `patternfly-addons.css` is imported here 
https://github.com/openshift/console/blob/master/frontend/public/components/cluster-settings/related-objects.tsx#L4

They are used in `/packages/ceph-storage-plugin/` and `/packages/local-storage-operator-plugin/` but I'm not sure where they are pulled in.